charts/argo-templates/scripts/query-non-null-rows.sh

```diff
@@ -34,7 +34,7 @@ declare -ra BQ_QUERY=(
   --replace=true
   --destination_table=${PROJECT}:${DATASET}.${TARGET_TABLE}
 )
-1>&2  ${BQ_QUERY[@]} "SELECT * EXCEPT (datarepo_row_id, ${REPO_KEYS})
+1>&2  ${BQ_QUERY[@]} "SELECT * EXCEPT (${REPO_KEYS})
```

Comment:
"datarepo_row_id" would be returned within "REPO_KEYS", right?

Reply:
Yup!